OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs

```diff
@@ -119,12 +119,13 @@ public DefaultSpriteSequence(ModData modData, TileSet tileSet, SpriteCache cache
 
 				var offset = LoadField<float2>(d, "Offset", float2.Zero);
 				var blendMode = LoadField<BlendMode>(d, "BlendMode", BlendMode.Alpha);
+				var alpha = LoadField<float>(d, "Alpha", 1f);
```

Comment:
This is not tied to any blend mode. OpenTK, OpenGL and PhotoShop seem to call it alpha (channel).